### ISSUE TITLE
Area not generated when lines have different orientation.

### ISSUE THREAD

**Issue by martinsluka** [therion/therion#365]
I tried to define area water defined by line floor-step and line border.

That area was not generated in final PDF. After I reversed orientation of line border, area was exported.
![Snímek obrazovky 2021-04-27 v 21 40 28](https://user-images.githubusercontent.com/7187070/116302995-1a236580-a7a2-11eb-8662-816db5f2db90.png)

**Comment by CaverBruce** [comment]
Yes, I noticed this effect the other day as well, and thought it strange that I had not noticed it as a problem before.
The work-arounds are not too hard though, and I was wondering if it was almost preferable behaviour.  But if it is a change and it makes old datasets fail, then it should probably be reinstated to behave as it used to.
